lib/scenic/statements.rb

```diff
@@ -21,7 +21,7 @@ module Statements
     #     SELECT * FROM users WHERE users.active = 't'
     #   SQL
     #
-    def create_view(name, version: nil, sql_definition: nil, materialized: false)
+    def create_view(name, version: nil, sql_definition: nil, materialized: false, materialized_no_data: false)
```

Comment:
I think we should consider going the route many rails migration statements take with extensions to options. So in my migration, I'd be able to do something like:

```ruby
create_view :search_results, version: 1, materialized: { no_data: true }
```

That would imply both materialized and no data. A view would be materialized if the option was present. That might require some gymnastics for reading the option, but I think the use is much clearer in that case. What do you think? cc @calebthompson